tests/meta/upload/test_meta_upload_scoutapi.py

```diff
@@ -3,35 +3,73 @@
 
 import pytest
 import yaml
+
 from cg.apps.hk import HousekeeperAPI
 from cg.meta.upload.scoutapi import UploadScoutAPI
 from cg.store import Store
 
 
-def test_generate_config_adds_rank_model_version(analysis: Store.Analysis, upload_scout_api:
-                                                 UploadScoutAPI):
+# pylint: disable=protected-access
```

Comment:
I would recommend moving protected members into a utility class if you want to test them :)

Reply:
I do not understand what you mean? I discussed this with @barrystokman before implementing. Perhaps you have another solution?

Reply:
Yes, in one place I created a small [class](https://github.com/Clinical-Genomics/cg/blob/master/cg/meta/report/sample_calculator.py) that is a public utility class and should be tested as such whereas private things are considered implementation details and it's not obvious how to test them without locking down the specific implementation..